frontend/src/components/dashboard/GalleryPreview.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import ImgMediaCard from "./ImgMediaCard";
+
+const useStyles = makeStyles((theme) => ({
+  galleryTitle: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'flex-start',
+    padding: theme.spacing(5)
+  },
+  viewMoreButton: {
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'flex-end',
+    padding: theme.spacing(4)
+  },
+  link: {
+    textDecoration: 'none',
+    color: theme.palette.text.primary
+  }
+}));
+
+export default function GalleryPreview(props) {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Container className={classes.galleryTitle}>
+        <Typography variant="h4">{props.name}</Typography>
```

Comment:
We may want to consider using h2 here (I assume the name of our website will eventually be placed as an h1) because as Nat mentioned in her ping earlier, it is important to use headers strategically.  When I was doing some research last week for the design doc I found[ this website ](https://webaccess.berkeley.edu/resources/tips/web-accessibility#accessible-CMS) and it mentions

>  "Be sure to adhere to the correct order of headings... Do not pick a header just because it looks good visually (which can confuse screen reader users); instead, create a new CSS class to style your text."

Reply:
Looks like this is an easy way to do it: https://material-ui.com/components/typography/#changing-the-semantic-element